docs: Fix invalid action subclass examples (#687)
krzema12/github-workflows-kt: docs/user-guide/using-actions.md

    library, i.e. a class that takes some constructor arguments with types of your choice, and maps them to strings
    inside `toYamlArguments`. Use it to have better type-safety when using the wrapper.

Inherit from [`Action`](https://github.com/krzema12/github-workflows-kt/blob/main/library/src/main/kotlin/it/krzeminski/githubactions/domain/actions/Action.kt)
and in case of actions without explicit outputs, use the `Actions.Outputs` class as type argument:

```kotlin
class MyCoolActionV3(
    private val someArgument: String,
) : Action<Action.Outputs>("acmecorp", "cool-action", "v3") {
    override fun toYamlArguments() = linkedMapOf(
        "some-argument" to someArgument,
    )

    override fun buildOutputObject(stepId: String) = Outputs(stepId)
}
```

> or, in case of actions with explicit outputs, create an own subclass of `Action.Outputs` for the type argument:

Reviewer: ```suggestion
or, in case of actions with explicit outputs, create a subclass of `Action.Outputs` for the type argument:
```
